data/initial-composition/slab-model/create_slab2_database.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This script uses the slab2 model data, available to download from
+# https://www.sciencebase.gov/catalog/item/5aa1b00ee4b0b1c392e86467, and
```

Comment:
I went to that website and there are different things that can be downloaded. It would be good to either provide the exact download link, or specify what needs to be downloaded.